deployments/helm/templates/fn-deployments.yaml

```diff
@@ -163,8 +163,10 @@ spec:
             - start
             - --home
             - /home/pd
-            - --host
-            - 0.0.0.0
+            - --grpc-bind
+            - 0.0.0.0:8080
+            - --metrics-bind
+            - 0.0.0.0:9000
```

Comment:
Post-merge I'll check on the preview deploy to confirm these new values are working as expected.